relay/kusama/src/lib.rs

```diff
@@ -1247,6 +1249,13 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
 				RuntimeCall::System(frame_system::Call::remark { .. }) |
 					RuntimeCall::System(frame_system::Call::remark_with_event { .. })
 			),
+			ProxyType::ParaRegistration => matches!(
+				c,
+				RuntimeCall::Registrar(paras_registrar::Call::reserve { .. }) |
+					RuntimeCall::Registrar(paras_registrar::Call::register { .. }) |
+					RuntimeCall::Utility(..) |
```

Comment:
Please match directly for `batch` etc calls and not all of the pallet.

Reply:
Made the changes!